r2dbc-mysql/src/main/java/JasyncStatement.kt

```diff
@@ -26,6 +26,7 @@ internal class JasyncStatement(private val clientSupplier: Supplier<JasyncConnec
 
     private var isPrepared = false
     private var selectLastInsertId: Boolean = false
+    private var releaseAfterwards: Boolean = false
```

Comment:
maybe call it releasePreparedStatementAfterUse (and also the method)?

Reply:
Thanks for the feedback! I've renamed both the variable and method